app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,7 @@
 <resources>
     <string name="app_name">Event Manager</string>
     <string name="title_activity_event_picking">EventPickingActivity</string>
+    <string name="picking_activity_help_text">Please select an event to continue.</string>
```

Comment:
Please be careful on naming convention:
  * `title_activity_event_picking`
  * `picking_activity_help_text`

... use somewhat reversed naming schemes. We have to choose one and stick to it.